common/djangoapps/auth_exchange/tests/test_forms.py

```diff
@@ -12,10 +12,12 @@
 from provider import scope
 import social.apps.django_app.utils as social_utils
 
-from auth_exchange.forms import AccessTokenExchangeForm
-from auth_exchange.tests.utils import AccessTokenExchangeTestMixin
 from third_party_auth.tests.utils import ThirdPartyOAuthTestMixinFacebook, ThirdPartyOAuthTestMixinGoogle
 
+from ..forms import AccessTokenExchangeForm
+from .utils import AccessTokenExchangeTestMixin
+from . import mixins
```

Comment:
No particular reason.  It seemed more straightforward at the time, but I see that it's against the prevailing, so I'll switch it for consistency.